src/main/java/com/hp/application/automation/tools/run/RunFromFileBuilder.java

```diff
@@ -542,7 +542,10 @@ public boolean isApplicable(
 		 * @return the job id
 		 */
 		@JavaScriptMethod
-		public String getJobId(String mcUrl, String mcUserName, String mcPassword, String proxyAddress, String proxyUserName, String proxyPassword) {
+		public String getJobId(String mcUrl, String mcUserName, String mcPassword, String proxyAddress, String proxyUserName, String proxyPassword, String previousJobId) {
```

Comment:
Please update JavaDoc to reflect changes.
It will be very helpful for future developers.